x-pack/plugins/ml/public/application/explorer/dashboard_controls/add_anomaly_charts_to_dashboard_controls.tsx

```diff
@@ -59,16 +63,19 @@ export const AddAnomalyChartsToDashboardControl: FC<AddToDashboardControlProps>
       };
     }
 
-    const config = getDefaultEmbeddablePanelConfig(jobIds);
+    const config = getDefaultEmbeddablePanelConfig(jobIds, queryString);
 
     return {
       ...config,
       jobIds,
       maxSeriesToPlot: maxSeriesToPlot ?? DEFAULT_MAX_SERIES_TO_PLOT,
       severityThreshold: severity.val,
-      ...(timeRange ?? {}),
+      ...(timeRange ? { timeRange } : {}),
```

Comment:
When adding to a dashboard, I don't think we should pass a time range, but default to the time range set in the dashboard. The user can then always set a custom time range for the panel, once in the dashboard, if they want:

<img width="361" alt="image" src="https://user-images.githubusercontent.com/7405507/189375812-da60821d-72e8-4939-b236-56d60adf261c.png">

Reply:
Updated here [`637eaa3` (#140147)](https://github.com/elastic/kibana/pull/140147/commits/637eaa3ad45965a9f5ef016d7942394f4ce2bd90)